CiscoVM/CiscoVM-1.0.0/ciscovm_reset.py

```diff
@@ -20,30 +20,16 @@
 SOFTWARE.
```

Comment:
This action allows to reset hash and status to reexport the device. It's going to be done monthly and on demand.